utils/generate-manifests.sh

```diff
@@ -7,8 +7,8 @@
 # 
 
 # For the Azure-Vote sample the following env varuables are supposed to be initialized: 
-# export VOTE_APP_TITLE='Awesome Voting App'
-# export AZURE_VOTE_IMAGE_REPO=gitopsflowacr.azurecr.io/azvote
+export VOTE_APP_TITLE='Awesome Voting App'
```

Comment:
We're setting this as part of the guide, shouldn't we leave these commented?

Reply:
We should leave these commented.